core/src/Platforms/android/LegacyCachePersistance.cs

```diff
@@ -71,8 +73,10 @@ void ILegacyCachePersistance.WriteCache(byte[] serializedCache)
                 }
                 catch (Exception ex)
             {
-                CoreLoggerBase.Default.Warning("Failed to save cache: " + ex.Message);
-                CoreLoggerBase.Default.ErrorPii(ex);
+                const string msg = "Failed to save cache: ";
```

Comment:
>Failed [](start = 36, length = 6)

... as above. consider adding adal or msal where relevant (seems like this is adal given the legacy naming

Reply:
done.